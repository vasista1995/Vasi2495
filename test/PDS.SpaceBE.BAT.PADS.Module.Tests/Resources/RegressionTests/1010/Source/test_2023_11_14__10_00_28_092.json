{
  "IdSource": "SPACEACT2:BAT:BE:RU342056B04:BE1I:Wire_Bond1:CopperBallHeight:67621:240185470:275",
  "Data_flat_Metadata": {
    "Device": "S0946T",
    "ParameterClass": "NON_SPC",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "CopperBallHeight",
    "ParameterUnit": "g",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "67621",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
    "CreatedTimestamp": "2023-11-14T13:17:22Z",
    "UpdatedTimestamp": "2023-11-14T14:46:31Z",
    "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
    "ExportedTimestampUtc": "2023-11-14T13:17:22Z",
    "ExportedTimestamp": "2023-11-14T13:17:22Z",
    "SampleTimestampUtc": "2023-11-14T06:17:22Z",
    "SampleTimestamp": "2023-11-14T13:17:22Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_Height/CU/50um",
    "RvStoreFlag": "Y",
    "CalcCreated": "01/25/2023 12:47:30",
    "ILdsID": 0,
    "SpcComExtern": "CA-Attributes started. Ticket id: 962d095c-48df-4c2d-a5a4-53eb58068805\nCA triggered by violation 1 (Raw value above specification) in channel (67621/275)\n====\nTicketId=962d095c-48df-4c2d-a5a4-53eb58068805 state=PASS executed OCAP=General, HOLDLOT, STOPEQUIPMENTPROCESSING, QUALITYALARMTRIGGER\nCA triggered by violation -1 (Unknown violation description) in channel (-1/-1)",
    "AcceptFlag": "Y",
    "UpdatedBy": "pobtech",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2138397,
    "ViolCount": 2,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "275",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 38.0,
        "SpecLow": 22.0,
        "SpecTarget": 30.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYNNNNNNNNNNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "38,08",
        "Median": "38",
        "Sigma": "1,45842380671738",
        "Min": "36,5",
        "Max": "39,7",
        "Q1": "36,8",
        "Q3": "39,4"
      },
      "ControlLimits": {
        "ctrl_mean_low": 23.6,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 30.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 36.4,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.0,
        "ctrl_range_high": 16.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 30.21575,
        "ExtEwmaMV": 32.4580418928263,
        "ExtMsMV": 6.74155605183254,
        "EwmaS": 2.30426778678397,
        "EwmaR": 5.27825829104797,
        "ExtEwmaMVLCL": 12.8656531239446,
        "ExtEwmaMVCenter": 15.268504,
        "ExtEwmaMVUCL": 17.6713548760554,
        "ExtMaMVLCL": 13.5694318513612,
        "ExtMaMVCenter": 15.268504,
        "ExtMaMVUCL": 16.9675761486388,
        "EwmaSLCL": 0.572474202225081,
        "EwmaSCenter": 1.51113357572865,
        "EwmaSUCL": 2.44979294923222,
        "EwmaRLCL": 1.38119890295338,
        "EwmaRCenter": 3.62764,
        "EwmaRUCL": 5.87408109704662,
        "ExtMSMVLCL": 0.00259778821504498,
        "ExtMSMVCenter": 1.2250389281796,
        "ExtMSMVUCL": 4.92102881621818,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 5.64136422726342,
        "RawUal": 24.8956437727366,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240185470,
        "Seqnr": 2,
        "Value": 36.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE"
      },
      {
        "SampleId": 240185470,
        "Seqnr": 3,
        "Value": 38.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE"
      },
      {
        "SampleId": 240185470,
        "Seqnr": 4,
        "Value": 36.5,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE"
      },
      {
        "SampleId": 240185470,
        "Seqnr": 5,
        "Value": 39.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE"
      },
      {
        "SampleId": 240185470,
        "Seqnr": 1,
        "Value": 39.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH\nCA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0\nTECH : DEDE"
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:CopperBallHeight:67621",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU342056B04:BE1I:Wire_Bond1:CopperBallHeight:67621:240185470:275"
  }
}