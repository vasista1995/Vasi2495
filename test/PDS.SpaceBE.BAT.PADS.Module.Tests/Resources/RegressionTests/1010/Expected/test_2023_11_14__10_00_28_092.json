{
  "Data_flat_Metadata": {
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T06:17:22Z",
    "BeginTimestamp": "2023-11-14T13:17:22Z",
    "EndTimestampUtc": "2023-11-14T06:17:22Z",
    "EndTimestamp": "2023-11-14T13:17:22Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "S0946T",
    "ParameterClass": "NON_SPC",
    "Segment": "WB"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 2,
          "SampleId": 240185470,
          "Value": 36.8,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:17:22Z",
          "SampleTimestampUtc": "2023-11-14T06:17:22Z",
          "CreatedTimestamp": "2023-11-14T13:17:22Z",
          "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
          "UpdatedTimestamp": "2023-11-14T14:46:31Z",
          "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "SampleMean": 38.08,
          "SampleMin": 36.5,
          "SampleMax": 39.7,
          "SampleMedian": 38.0,
          "SampleStdev": 1.45842380671738,
          "SampleQ1": 36.8,
          "SampleQ3": 39.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 240185470,
          "Value": 38.0,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:17:22Z",
          "SampleTimestampUtc": "2023-11-14T06:17:22Z",
          "CreatedTimestamp": "2023-11-14T13:17:22Z",
          "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
          "UpdatedTimestamp": "2023-11-14T14:46:31Z",
          "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "SampleMean": 38.08,
          "SampleMin": 36.5,
          "SampleMax": 39.7,
          "SampleMedian": 38.0,
          "SampleStdev": 1.45842380671738,
          "SampleQ1": 36.8,
          "SampleQ3": 39.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 240185470,
          "Value": 36.5,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:17:22Z",
          "SampleTimestampUtc": "2023-11-14T06:17:22Z",
          "CreatedTimestamp": "2023-11-14T13:17:22Z",
          "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
          "UpdatedTimestamp": "2023-11-14T14:46:31Z",
          "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "SampleMean": 38.08,
          "SampleMin": 36.5,
          "SampleMax": 39.7,
          "SampleMedian": 38.0,
          "SampleStdev": 1.45842380671738,
          "SampleQ1": 36.8,
          "SampleQ3": 39.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 240185470,
          "Value": 39.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:17:22Z",
          "SampleTimestampUtc": "2023-11-14T06:17:22Z",
          "CreatedTimestamp": "2023-11-14T13:17:22Z",
          "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
          "UpdatedTimestamp": "2023-11-14T14:46:31Z",
          "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "SampleMean": 38.08,
          "SampleMin": 36.5,
          "SampleMax": 39.7,
          "SampleMedian": 38.0,
          "SampleStdev": 1.45842380671738,
          "SampleQ1": 36.8,
          "SampleQ3": 39.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 1,
          "SampleId": 240185470,
          "Value": 39.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:17:22Z",
          "SampleTimestampUtc": "2023-11-14T06:17:22Z",
          "CreatedTimestamp": "2023-11-14T13:17:22Z",
          "CreatedTimestampUtc": "2023-11-14T06:17:22Z",
          "UpdatedTimestamp": "2023-11-14T14:46:31Z",
          "UpdatedTimestampUtc": "2023-11-14T07:46:31Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
          "SampleMean": 38.08,
          "SampleMin": 36.5,
          "SampleMax": 39.7,
          "SampleMedian": 38.0,
          "SampleStdev": 1.45842380671738,
          "SampleQ1": 36.8,
          "SampleQ3": 39.4,
          "SampleSize": 5
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "275",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 38.0,
          "spec_low": 22.0,
          "target": 30.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 23.6,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 30.0,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 36.4,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 0.0,
          "ctrl_range_high": 16.0,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 38.08,
        "median": 38.0,
        "stdev": 1.3044539087296272,
        "range": 3.200000000000003,
        "min": 36.5,
        "max": 39.7,
        "q02": 36.524,
        "q05": 36.56,
        "q25": 36.8,
        "q75": 39.4,
        "q95": 39.64,
        "q98": 39.676,
        "NumViolations": 2,
        "ViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
        "ViolationList": "1, 3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO  TIME OFFSET TO HIGH CA : ADJUST EFO TIME OFFSET FROM 0.2 TO 0.0 TECH : DEDE",
        "Samples": "240185470"
      },
      "ParameterName": "CopperBallHeight",
      "ParameterUnit": "g",
      "ChannelId": "67621",
      "SourceDataLevel": "C",
      "channel_name": "Ball_Height CU 50um",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T06:17:22Z",
      "created_timestamp": "2023-11-14T13:17:22Z",
      "updated_timestamp_utc": "2023-11-14T07:46:31Z",
      "updated_timestamp": "2023-11-14T14:46:31Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T07:46:31Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU342056B04:BE1I:Wire_Bond1:CopperBallHeight:67621:240185470:275",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T13:17:22Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}